### DISCUSSION TITLE
Envoy Advanced Configuration Include watched_directory field

### DISCUSSION THREAD

**Discussion by zachhutchins** [mia-platform/community#486] [Ideas]
### Which area of this project could be most improved?

Console

### Description

In Mia Api-gateway advanced configuration or anywhere else that is appropriate, if we could specify the envoy configuration file to include `watched_directories` option.

This kind of configuration would allow us to take advantage of auto refresh of configuration changes to envoy.

https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/config_source.proto#config-core-v3-pathconfigsource

**Comment by galatifederico** [comment]
Hi @zachhutchins, in the Mia Platform's current setup for Envoy advanced configuration, the ability to specify the watched_directories option in the Envoy configuration file is not available. 

We will keep you updated through the support team on any future developments.
